chapter2.adoc

```diff
@@ -1,214 +1,207 @@
 [[Sdsec]]
 == Sdsec (ISA extension)
 
-This chapter introduces the Sdsec ISA extension, which extends the ISA section of The RISC-V Debug Specification. The RISC-V hart must implement all features to ensure external debug security. It is designed to enforce access control over operations initiated by the Debug Module, as well as constraints on trigger behaviors cite:[dbgspec]. Additionally, it incorporates trace functionality cite:[etrace], with its output undergoing check based on hart privilege levels.
+This chapter introduces Sdsec ISA extension, which enhances the Sdext of _The RISC-V Debug Specification_ cite:[dbgspec]. The Sdsec extension provides privilege based protection for debug operations and trigger behaviors cite:[dbgspec]. Furthermore, it constrains trace functionality cite:[etrace] according to RISC-V privilege levels. 
```

Comment:
This paragraph summarizes the affected extension. To provide a complete picture, it might be helpful to reference explicitly Sdtrig here.

Reply:
sure, we can do it.